Makefile

```diff
@@ -28,31 +30,49 @@ ifeq ($(RPM_ARCH),)
     RPM_ARCH = $(shell uname -m)
 endif
 
-# DEB build variables
-DEB_DIST ?=
+# Debian Packaging
+DEBUILD_BIN ?= debuild
+DEBUILD_OPTS = --source-option="-I"
+DPUT_BIN ?= dput
+DPUT_OPTS ?=
+DEB_DIST ?= xenial
+
+ifeq ($(origin GPG_SIGNING_KEY), undefined)
+    GPG_SIGNING_KEY = /dev/null
+endif
+
+ifeq ($(OFFICIAL),yes)
+    # Sign official builds
+    DEBUILD_OPTS += -k$(GPG_KEY_ID)
+else
+    # Do not sign unofficial builds
+    DEBUILD_OPTS += -uc -us
+endif
+
+DEBUILD = $(DEBUILD_BIN) $(DEBUILD_OPTS)
+DEB_PPA ?= mini_dinstall
 DEB_ARCH ?= amd64
 DEB_NVR = $(NAME)_$(VERSION)-$(RELEASE)~$(DEB_DIST)
 DEB_NVRA = $(DEB_NVR)_$(DEB_ARCH)
+DEB_NVRS = $(DEB_NVR)_source
 DEB_TAR_NAME=$(NAME)-$(VERSION)
 DEB_TAR_FILE=$(NAME)_$(VERSION).orig.tar.gz
-DEBUILD_BIN ?= debuild
-DEBUILD_OPTS ?= -us -uc
-DEBUILD = $(DEBUILD_BIN) $(DEBUILD_OPTS)
-
+DEB_DATE := $(shell LC_TIME=C date +"%a, %d %b %Y %T %z")
 
-.PHONY: clean dist sdist dev shell image devimage rpm srpm docs
+.PHONY: clean dist sdist dev shell image devimage rpm srpm docs deb debian deb-src
 
 clean:
 	rm -rf dist
+	rm -rf ansible-runner.egg-info
 	rm -rf rpm-build
 	rm -rf deb-build
 
 dist:
 	$(DIST_PYTHON) setup.py bdist_wheel --universal
 
-sdist: dist/$(PIP_NAME)-$(VERSION).tar.gz
+sdist: dist/$(NAME)-$(VERSION).tar.gz
```

Comment:
It seems like setup.py is creating this file as `ansible-runner`, not `ansible_runner`, so we have to use `$(NAME)` here, not `$(PIP_NAME)`.  But if that's true, how did it ever work?  So there's clearly something I don't understand going on here.

Reply:
My vague understanding of Make suggests that it doesn't care much what you call a rule, but if a file of the same name doesn't exist it'll always run instead of skipping when the artifact of that rule is up to date.

Reply:
I think you're exactly right.  I think it was working before because the `PIP_NAME` version of the filename was just a rule name, that didn't actually match the files being generated, but who cares because Make will still chain to the next rule.